gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,6 @@
-#Wed Jun 27 14:32:00 EDT 2018
+#Wed Oct 30 21:16:00 EET 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
 distributionUrl=https\://services.gradle.org/distributions/gradle-4.8.1-all.zip
-distributionSha256Sum=ce1645ff129d11aad62dab70d63426fdce6cfd646fa309dc5dc5255dd03c7c11

```

Comment:
Not needed

Reply:
Got this warning in Android Studio 

WARNING: It is not fully supported to define distributionSha256Sum in gradle-wrapper.properties.
Using an incorrect value may freeze or crash Android Studio.
Please manually verify or remove this property from all of included projects if applicable.
For more details, see https://github.com/gradle/gradle/issues/9361.

Use "ce1645..." as checksum for https://services.gradle.org/distributions/gradle-4.8.1-all.zip and sync project
Remove distributionSha256Sum and sync project
Open Gradle wrapper properties

Reply:
I really don't care about Android Studio bugs: Please keep the checksum around